src/ui/public/notify/notifier.js

```diff
@@ -363,12 +394,57 @@ Notifier.prototype.banner = function (msg, cb) {
   return this.set({
     type: 'banner',
     title: 'Attention',
-    markdown: formatMsg(msg, this.from),
+    content: formatMsg(msg, this.from),
     lifetime: Notifier.config.bannerLifetime,
     actions: ['accept']
   }, cb);
 };
 
+/**
+ * Display a custom message
+ * @param  {String} msg - required
+ * @param  {Object} config - required
+ * @param  {Function} cb - optional
+ *
+ * config = {
+ *   title: 'Some Title here',
+ *   type: 'info',
+ *   actions: [{
+ *     text: 'next',
+ *     callback: function() { next(); }
+ *   }, {
+ *     text: 'prev',
+ *     callback: function() { prev(); }
+ *   }]
+ * }
+ */
+Notifier.prototype.custom = function (msg, config, cb) {
+  // There is no helper condition that will allow for 2 parameters, as the
+  // other methods have. So do a type check that config is an object
+  const isObject = (value) => _.isObject(value) && !_.isArray(value);
+  if (!isObject(config)) {
+    throw new Error('config param is required, and must be an object');
+  }
+
+  const customActionMax = 3;
+  const mergedConfig = _.assign({
+    type: 'custom',
+    title: 'Notification',
+    content: formatMsg(msg, this.from),
+    lifetime: Notifier.config.infoLifetime,
+  }, config);
+
+  const hasActions = _.get(mergedConfig, 'actions.length');
+  if (!hasActions) {
+    mergedConfig.actions = ['accept'];
+  } else {
+    mergedConfig.customActions = mergedConfig.actions.slice(0, customActionMax);
```

Comment:
If I'm reading this right, this code will take the first 3 custom actions and use them, discarding any others. As a caller of this code, this could be confusing if I pass in more than 3 custom actions. What do you think about throwing an error if more than `customActionMax` actions are passed in?